workshop-2/workshop_2.ino

```diff
@@ -0,0 +1,66 @@
+
+                                     //WORKSHOP 2//
+                            //YNOV - Informatique ingé-sup//
+                 //Julien KHABBAZ - Guillaume FAYET - Quentin FAYET//
+                                 //year 2018 - 2019//
+
+                                                             //affectation des pin
+const byte ledPin           = 10;      
+const byte interruptPin     = 2;
+const int  potentiometerPin = A0;
+
+//affectation des variable
+int           incomingByte = 0; 
+volatile byte state = LOW;
```

Comment:
As you make some boolean operation on blink function like `state != state;` state should be a bool